### ISSUE TITLE
pip python-bitcoinrpc is not up-to-date

### ISSUE THREAD

**Issue by pinheadmz** [jgarzik/python-bitcoinrpc#62]
Specifically is missing this commit:

https://github.com/jgarzik/python-bitcoinrpc/commit/8c0114bfbf7650d40a88b20d1e16ff79d768f3a9

My experience:
`sudo pip install python-bitcoinrpc`

Then running this script:
```
from bitcoinrpc.authproxy import AuthServiceProxy, JSONRPCException
rpc_connection = AuthServiceProxy("http://%s:%s@127.0.0.1:8332"%("USER","PW"))
print(rpc_connection.getinfo())
```
simply outputs:
`None`

After editing /usr/local/lib/python2.7/dist-packages/bitcoinrpc/authproxy.py manually removing the "else" in line 146, the same script produces better results:
```
{u'connections': 10, u'errors': u'', u'blocks': 393219, u'paytxfee': Decimal('0E-8'), u'keypoololdest': 1451977259, u'walletversion': 60000, u'difficulty': Decimal('113354299801.47113037'), u'testnet': False, u'version': 110000, u'proxy': u'', u'protocolversion': 70010, u'timeoffset': -1, u'balance': Decimal('0E-8'), u'unlocked_until': 0, u'relayfee': Decimal('0.00005000'), u'keypoolsize': 101}
```

**Comment by shivaenigma** [comment]
Can this be fixed.  This is creating lot of confusion http://bitcoin.stackexchange.com/questions/42134/python-bitcoinrpc-only-debug-available

**Comment by ShawnKimble** [comment]
Bumping this for priority.  Being out of the loop, its very confusing to resolve and identify the "NONE" issue as a repository bug.

**Comment by nmarley** [comment]
I got this too -- is there any way to bump the version and update the PyPI repository?

What needs to be done to push this to the repo? @jgarzik ?

**Comment by jgarzik** [comment]
I'm not the maintainer of the pip/PyPi repo - no linkage with this github.

Happy to bump the version in this repo, if that helps.

**Comment by nmarley** [comment]
Ok, thanks Jeff. Guess I'll have to dig thru there and see who that is then.

Can you please bump the version? I submitted a PR #68 to bump the version if that makes it easier for you, or however you wanna do it.

**Comment by jgarzik** [comment]
Fixed in 76ced424dc16f997365265487487056e653238c6

**Comment by nmarley** [comment]
Emailed the maintainer and he pushed the latest version (1.0), so this is now resolved.
